HISTORY.rst

```diff
@@ -1,11 +1,17 @@
 # History
 
-2.0.0-b1
+2.0.0-b0 (2022-11-22)
```

Comment:
1 header is enough, only the last date will do. This is for referring back to see what changed between releases

Reply:
Just wanted to make clear what's in the PR.